_posts/national/united-kingdom/united-kingdom.Rmd

```diff
@@ -62,3 +61,121 @@ summary_path <- file.path("covid-rt-estimates/subnational", folder, "cases/summa
 
 ```{r,  child = system.file("templates/_regional-summary.Rmd", package = "EpiNow2")}
 ```
+
+## Comparison of cases, admissions, and deaths data
```

Comment:
is this in the correct order? I would have thought this would replace the templates or at least come before?